app/services/ucas_matches/send_resolved_on_apply_emails.rb

```diff
@@ -0,0 +1,21 @@
+module UCASMatches
+  class SendResolvedOnApplyEmails
+    attr_reader :ucas_match
+
+    def initialize(ucas_match)
+      @ucas_match = ucas_match
+    end
+
+    def call
+      raise 'The application has not been resolved on Apply' if !ucas_match.resolved_on_apply?
```

Comment:
I think in this instance a negated if reads better, eg raise an error if the match has not been resolved on apply rather than raise an error unless it has been resolved.

Reply:
yeah, I agree it reads nicer. I used to be told off by rubocop for `if !`